### DISCUSSION TITLE
Preserve React Table Sorting,Pagination, Filtering

### DISCUSSION THREAD

**Discussion by ramyadaggula** [TanStack/table#2713] [General]
Hi,

I have a page, which has 3 tabs component on it. When each tab is selected there is related ReactTable, each row on the table has Action button, when clicked it opens a SharePoint dialog modal when closing the dialog modal the page refreshes and all the sorting, filtering settings are gone. User requests to preserve their settings not reset them on page refresh. How to save them to localStorage variable. Any help is much appreciated, Thank you! .

**Comment by 0-don** [comment]
yes this would be nice addition
